templates/CRM/Contact/Form/Contact.hlp

```diff
@@ -128,6 +128,13 @@
 <p>{ts}Upload a photo or icon that you want to be displayed when viewing this contact.{/ts}</p>
 {/htxt}
 
+{htxt id="id-contact-id-title"}
+  {ts}Contact ID{/ts}
+{/htxt}
+{htxt id="id-contact-id"}
+<p>{ts}Every contact in CiviCRM has a unique ID number. This number will never change and is the most accurate way of identifying a contact.{/ts}</p>
+{/htxt}
+
```

Comment:
@reecebenson 
Don't we have same code at https://github.com/civicrm/civicrm-core/pull/13568/files#diff-82ac4c4ee8b147cbe480496609dcf0c7R139